create_deployments.sh

```diff
@@ -1,12 +1,14 @@
 export $(grep -v '^#' .env | xargs)
 
-prefect deployment build ./orchestration/flows/bl832/move.py:process_new_832_file -n new_file_832 -q bl832
-prefect deployment apply process_new_832_file-deployment.yaml
+prefect deployment build ./orchestration/flows/bl7012/move.py:process_new_file_ptycho4 -n 'process_newdata7012_ptycho4' -q bl7012_ptycho4
+prefect deployment apply process_new_file_ptycho4-deployment.yaml
 
+prefect deployment build ./orchestration/flows/bl7012/move_recon.py:transfer_auto_recon -n 'transfer_auto_recon' -q transfer_auto_recon
+prefect deployment apply transfer_auto_recon-deployment.yaml
 
-prefect deployment build ./orchestration/flows/bl832/move.py:test_transfers_832 -n test_transfers_832 -q bl832
```

Comment:
Let's separate these scripts by beamline. This change removes one of the deployments from the 8.3.2 set.